### ISSUE TITLE
Question related to the 6.1.21 kernel version

### ISSUE THREAD

**Issue by rack570** [nexmonster/nexmon_csi_bin#7]
Hello? I'm wondering if the nexmon_csi_bin has been updated to work with kernel version 6.1.21. Could you share the file path for the 6.1.21 kernel version? I tried installing it on the OS released on May 3, 2023, but it doesn't seem to work well.
Could it be that the mcp command is working, but the error is coming from nexutil? Here's the error output.

nex_init_netlink: socket error (93: Protocol not supported)
nex_init_netlink: socket error (93: Protocol not supported)
nex_init_netlink: bind error (9: Bad file descriptor)
nex_init_netlink: connect error (9: Bad file descriptor)
ERR (__nex_driver_netlink): no valid answer received
command failed: Operation not supported (-95)
Cannot find device "mon0"

**Comment by zeroby0** [comment]
Yes, it doesn't seem to work well. I'll take another try at it again when I have some time
